librosa/core/spectrum.py

```diff
@@ -92,7 +93,7 @@ def stft(y, n_fft=2048, hop_length=None, win_length=None, window='hann',
     --------
     istft : Inverse STFT
 
-    ifgram : Instantaneous frequency spectrogram
+    reassigned_spectrogram : Time-frequency reassigned STFT
```

Comment:
STFT magnitudes after time-frequency reassignment


(time-frequency is established in the literature as an adjective, not an adverb. also it's unclear that phases are no longer available)

Reply:
Er, reassigned_spectrogram does return phase?

Reply:
!! you're right, nevermind.

Why did we do that?  Shouldn't reassigned_spectrogram return the stft instead of magnitudes?

Reply:
no, it doesn't. which is why we call it a "reassigned spectrogram", not a "reassigned STFT".
my suggestion is to insist on the fact that reassigned_spectrogram returns STFT magnitudes, not complex-valued coefficients

Reply:
> my suggestion is to insist on the fact that reassigned_spectrogram returns STFT magnitudes, not complex-valued coefficients

We could also just change it to return complex valued coefficients, so that feature parity is maintained with the deprecated ifgram.  Since it hasn't been released yet, we still have some flexibility to change things.

I guess I really dropped the ball on CR for that one -- is it standard for TFR to produce magnitudes instead of proper STFTs?  I can't imagine a good reason for that, but maybe my imagination isn't very good.

Reply:
Yes, it's standard for time-frequency reassignment to only return the magnitudes. The phase is ill-defined after time-frequency reassignment. The TFR is not a bilinear time-frequency distribution in the sense of the Cohen's class because its smoothing kernel with respect to the Wigner-Ville is not positive.

Reply:
Ok, TIL! Thanks for clarifying.